doc/api/console.md

```diff
@@ -277,6 +277,21 @@ Defaults to `2`. To make it recurse indefinitely, pass `null`.
 Defaults to `false`. Colors are customizable; see
 [customizing `util.inspect()` colors][].
 
+### console.dirxml(...data)
+<!-- YAML
+added: v8.0.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/17128
+    description: "`console.dirxml` now calls `console.dir` for each argument."
+-->
+* `...data` {any}
+
+This method calls `console.dir()` with default options for each argument it
+receives. See [`console.dir()`][] for more details about said defaults.
+Please note that this method doesn't produce any xml formatting and uses the
+default `console.dir()` formatting resolution instead.
```

Comment:
Not a native English speaker myself, but `resolution` sounds strange here IMO (at least I never used it in this context). It is not really necessary anyway as the first sentence already has the same information.

Reply:
Well, I'll just let the precision on the absence of XML formatting then, juste to be sure it's clear for everyone. But I thought resolution was indicated, you have me doubting now 😄